cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -485,6 +486,8 @@ public static final class SystemWorker {
     public static final String DISPATCH_PROGRAM_TYPES = "system.worker.dispatch.program.types";
     public static final String HTTP_CLIENT_READ_TIMEOUT_MS = "system.worker.http.client.read.timeout.ms";
     public static final String HTTP_CLIENT_CONNECTION_TIMEOUT_MS = "system.worker.http.client.connection.timeout.ms";
+    public static final String TWILL_CONTROLLER_START_SECONDS = "system.worker.app.program.twill.controller.start" +
```

Comment:
nit: a little less verbose to name it system.worker.program... instead of system.worker.app.program